firebase/functions/src/types/collections/NotificationsCollectionEntry.ts

```diff
@@ -0,0 +1,13 @@
+export interface NotificationsCollectionEntry {
+  d: {
+    uid: string
+    timestamp: string
```

Comment:
should be an integer
```suggestion
    timestamp: number
```